.github/workflows/macos-build.yml

```diff
@@ -20,6 +20,10 @@ jobs:
       uses: actions/checkout@v2
       with:
         path: FStar
+    - name: Install .NET SDK
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: '6.0.x'
```

Comment:
I believe this change here is indeed needed, because tests on Mac OS X are not Dockerized. Could anyone on Mac please confirm?

Reply:
@tahina-pro : Are we aware of anyone using macos builds?